packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-services.4/SeleniumDockerService.tool.mdx

```diff
@@ -1,39 +1,58 @@
+import { Notice } from '@cerner/terra-docs';
+
 # Selenium Docker Service
 
-The selenium docker service initializes a docker swarm for running functional [WebDriverIO](https://webdriver.io/) tests against a containerized [selenium docker](https://github.com/SeleniumHQ/docker-selenium) environment.
+The selenium docker service initializes a docker-compose for running functional [WebDriverIO](https://webdriver.io/) tests against a containerized [selenium docker](https://github.com/SeleniumHQ/docker-selenium) environment.
 
-By default, the selenium grid will be deployed locally on the host machine running on port 4444. Google Chrome and Firefox browsers are available on the grid.
+By default, the selenium grid will be deployed locally on the host machine running on port 4444. Google Chrome and Firefox are available on the docker container. Internet Explorer can be enable when using a remote selenium grid that has it available.
 
-## Installation
+## Options
 
-The selenium docker service should be installed as a development dependency.
+The selenium docker service options are configured via the [test runner CLI options](../about#options).
 
-```bash
-npm install --save-dev @cerner/terra-functional-testing
-```
+### disableSeleniumService
+
+A flag to disable the selenium docker service for the test run.
+
+Type: `bool`
+
+Required: `false`
+
+Default: `false`
 
-## Configuration
+Example:
 
-To use the service it must be added to the services array in the `wdio.conf.js` file.
+<Notice variant="important" ariaLevel="3">
+  The selenium docker service is disabled automatically by the test runner if a remote selenium grid is specified.
+</Notice>
 
 ```js
-// wdio.conf.js
-const SeleniumDockerService = require('@cerner/terra-functional-testing/lib/services/wdio-selenium-docker-service');
-
-export.config = {
-    // ...
-    services: [
-        [SeleniumDockerService]
-    ],
-    // ...
-};
+"scripts": {
+  "test:wdio": "terra wdio --disableSeleniumService"
+}
 ```
 
-## Options
+### keepAliveSeleniumDockerService
+
+A flag to keep the selenium docker service running upon test completion.
```

Comment:
Given the change on #582 do we still want to list this option since there is very little benefit in using it? If it is worth having it here we should elaborate on when/why to use this option. Something like when multiple tests are executed sequentially in a singe script command this option can be used to keep the selenium docker service alive and not need to stop the service in between test runs.

Should we show multiple test runs in the example below?
`"test:wdio": "terra wdio --keepAliveSeleniumDockerService && terra wdio --themes orion-fusion-theme"`

Reply:
maybe for now we just drop it from documentation... i could mostly see people getting in trouble trying to use it.

Reply:
Dropped for now:

https://github.com/cerner/terra-toolkit/pull/610/commits/36606851682a8a89e5cc76e7a6ec39436bd45d0c